Casks/eddie.rb

```diff
@@ -1,16 +1,24 @@
 cask "eddie" do
   version "2.20.0"
+  
+  if Hardware::CPU.intel?
+  
+    if MacOS.version <= :mojave
+      sha256 "a446563a6beb6f91542d6afbd6f90f6745b7a36fef363a13506cff9d8f078c78"
 
-  if MacOS.version <= :mojave
-    sha256 "a446563a6beb6f91542d6afbd6f90f6745b7a36fef363a13506cff9d8f078c78"
+      url "https://eddie.website/download/?platform=macos-10.9&arch=x64&ui=ui&format=disk.dmg&version=#{version}",
+          verified: "eddie.website/"
+    else
+      sha256 "3f960bfd47ef5847197012bf0455003f6b4999b9b6c52e58f61e8d5c2e81d263"
 
-    url "https://eddie.website/download/?platform=macos-10.9&arch=x64&ui=ui&format=disk.dmg&version=#{version}",
-        verified: "eddie.website/"
+      url "https://eddie.website/download/?platform=macos-10.15&arch=x64&ui=ui&format=disk.dmg&version=#{version}",
+          verified: "eddie.website/"
   else
```

Comment:
@adi1382 You will need to close the second `if` statement.

```suggestion
    end
  else
```